css/style.css

```diff
@@ -5,26 +5,75 @@ body {
 	background-color: black;
 }
 
-ul.thumbnails {
-	color: white;
+/*-make bg image separate layer
+	-make black overlay 
+	-put content on top
+*/
+
+div.bg-img {
+	position: absolute;
+	z-index: 10;
+
+	top: 0;
+	right: 0;
+	left: 0;
+	bottom: 0;
+	background-image: url("../img/photos/kitchen3.jpg");
+	background-size: cover;
+	background-position: center center;
 }
 
+div.black-overlay {
+	position: absolute;
+	z-index: 20;
 
-
-
-
-
-
+	opacity: .3;
+	top: 0;
+	left: 0;
+	right: 0;
+	bottom: 0;
+	background-color: black;
+}
 
 div.hero-unit{
 	padding-top: 30px;
 	padding-bottom: 270px;
+	position: relative;
 }
 
```

Comment:
Maybe this part can be refactored? I guess this is what was necessary without touching boostrap's defined class for that hero-unit section. Keep for now.